### PR TITLE
Add hand tracking documention to OpenXR chapter

### PR DESCRIPTION
**Pull request by BastiaanOlij**
This details out the hand tracking support that is available in our Godot 3 OpenXR plugin.

Related to: https://github.com/GodotVR/godot_openxr/issues/159

### CODE REVIEW

File: tutorials/vr/openxr/handtracking.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+At this point in time the plugin only exposes the data as it is provided by the OpenXR runtime. The plugin has an example implementation based on meshes that Valve has made publicly available however these work most reliable when used in conjunction with SteamVR.
+These scenes are `scenes/left_hand_mesh.tscn` and `scenes/right_hand_mesh.tscn` and can be childed to the :ref:`ARVROrigin <class_ARVROrigin>` node.
+
+.. image:: img/arvr_mesh_example.png
+
+Below is an overview of the steps needed to implement your own version.
+
+.. note::
+    The best way to implement this logic is to ask an artist to model a hand in 3D software using a real hands dimentions and create an armature for the hand that follows the bone structure exactly as the OpenXR specification dictates in the image at the top of this article. When skinning special care needs to be taken keeping in mind that if full hand tracking is available, the distance between joints will be determined by the actual size of the players hand and may thus be different to the 3D model. After importing the model into Godot you can add the required scripts to make everything work.
```
Comment (m4gr3d) [inline tutorials/vr/openxr/handtracking.rst:76]: `using real hands dimensions` instead of `using a real hands dimentions`.

**Review by m4gr3d — APPROVED** [review]

File: tutorials/vr/openxr/handtracking.rst (hunk 1 above)
Comment (BastiaanOlij) [reply tutorials/vr/openxr/handtracking.rst]: cheers

**Review by mhilbrunner — APPROVED** [review]
Read it, approved. :) Good work. Does this only apply to 3.4?

**Comment by m4gr3d** [comment]
> Read it, approved. :) Good work. Does this only apply to 3.4?

Yes, `3.4` (and `stable`).

**Comment by mhilbrunner** [comment]
Thanks!

**Pull request merged**